src/extensions/cluster-feature-registry.ts

```diff
@@ -0,0 +1,23 @@
+import { observable } from "mobx"
+import { Feature } from "../main/feature";
+
+export interface ClusterFeatureRegistration {
+  title: string;
+  description: string;
+  feature: Feature
+}
+
+export class ClusterFeatureRegistry {
```

Comment:
Please have a look file `base-registry.ts` to avoid some boilerplate (this file added in my previous PR, it's already re-used with some stores)

Reply:
Refactored to use `BaseRegistry`.